src/App/Hovedside/Hovedside.tsx

```diff
@@ -7,8 +12,25 @@ import ArbeidsgiverTelefon from "./ArbeidsgiverTelefon/ArbeidsgiverTelefon";
 import KontaktOss from "./KontaktOss/KontaktOss";
 import AltinnBoks from "./AltinnBoks/AltinnBoks";
 import { pamLink, syfoLink } from "../../lenker";
+import { sjekkPamTilgang } from "../../api/pamApi";
+import { OrganisasjonContext } from "../../OrganisasjonProvider";
 
 const Hovedside: FunctionComponent = () => {
+  const [tilgangTilPam, setTilgangTilPam] = useState(false);
+  const { valgtOrganisasjon } = useContext(OrganisasjonContext);
+
+  useEffect(() => {
+    const hentPamTilgang = async () => {
```

Comment:
Er det noen spesiell grunn til at dette wrappes inn i funksjonen "hentPamTIlgang"?

Reply:
UseEffect har ikke lov til å være async

Reply:
Man kan ikke markere arrow funksjonen til useEffect som async. Så man må gjøre det på dette viset... Mulig det står noe smart i denne 49 minutters artikkelen https://overreacted.io/a-complete-guide-to-useeffect/